wp-content/mu-plugins/pub/wporg-learn-cli.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+if ( ! defined( 'WP_CLI' ) ) {
+	return;
+}
+
+/**
+ * Converts a tutorial post to a lesson post type.
+ */
+class WPORG_Learn_Tutorial_To_Lesson_Command extends WP_CLI_Command {
+	/**
+	 * Converts tutorials to lessons based on the provided URL or file.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <source>
+	 * : The URL of the tutorial post to convert, or path to a file containing URLs (one per line)
+	 *
+	 * [--live]
+	 * : Actually perform the conversion (default is dry-run)
```

Comment:
in most cases having a `dry-run` variable is how it's normally done, but this is a minor nitpick and not a blocker.